core/src/i18n/en.yaml

```
@@ -167,6 +167,24 @@ deployments:
   title: Deployments
 discovery-and-network:
   title: Discovery and Network
+dnsentries:
+  headers:
+    dns-name: DNS Name
+    provider: Provider
+    provider-type: Provider type
+    spec: Specification
+    status: $t(common.headers.status)
+    targets: Targets
+    ttl: TTL
+  title: DNS Entries
+dnsproviders:
+  domains: Domains
+  headers:
+    exclude: Excluded domains
```

here, too, capitalization might be better

```suggestion
    exclude: Excluded Domains
```